### ISSUE TITLE
Fatal Error: Call to a member function logAddLogger() AND include once errors after update to 3.7

### ISSUE THREAD

**Issue by gocha-gochitashvili** [joomla/joomla-cms#15827]
Steps to reproduce the issue

Update Joomla! to version 3.7 from 3.6.5

Expected result

Everything is fine

Actual result

1) Fatal error: Call to a member function logAddLogger() on null in ...../libraries/fof/include.php
2) include_once( '/usr/www/users/...../includes/framework.php' )	.../api.php:9
2) require_once( '/usr/www/users/...../libraries/cms.php' )	.../framework.php:23
2) include_once( '/usr/www/users/...../libraries/fof/include.php' )
2) System information (as much as possible)

Additional comments

We have api.php and in 3.6.5 worls perfect, but when update we get this messages. 
included files in our api:

```
define('_JEXEC', 1);
define('JPATH_BASE', realpath('../'));
define('KPATH_ADMIN', JPATH_BASE . '/administrator/components/com_kunena');
ini_set('display_errors', '1');
error_reporting(1);
define('JDEBUG', 1);
include_once JPATH_BASE . '/includes/defines.php';
include_once JPATH_BASE . '/includes/framework.php';
include_once JPATH_BASE . '/libraries/joomla/factory.php';
include_once JPATH_BASE . '/components/com_community/libraries/core.php';
include_once JPATH_BASE . '/components/com_community/libraries/messaging.php';
include_once JPATH_BASE . '/libraries/kunena/bootstrap.php';
include_once JPATH_BASE . '/components/com_kunena/models/category.php';
```

**Comment by PhilETaylor** [comment]
Please test PR #15843

**Comment by franz-wohlkoenig** [comment]
closed as having PR #15843<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15827">issues.joomla.org/tracker/joomla-cms/15827</a>.</sub>
